docs: add info on modifying and adding utilities (#31328)
twbs/bootstrap: site/content/docs/5.0/utilities/api.md

  "float": null,
);
```

## Adding to existing utilities

> The default utilities can be added to with `map-merge`
> 
> Make sure to import this file instead of the default file in your Bootstrap imports

Reviewer: ```suggestion
Add new utilities with `map-merge`. Start by importing the utilities stylesheet, then use `map-merge` to add new properties and values.
```

```scss

> @import "[PATH TO BOOTSTRAP]/scss/utilities";

Reviewer: ```suggestion
@import "bootstrap/scss/utilities";
```

$utilities: map-merge(
  $utilities,
  (
    "font-size": (
      responsive: true,
      property: font-weight,
      values: $display-font-sizes,
    )
  )
);
```

## Modifying existing utilities

> The default utilities can be modified with a combination `map-get` and `map-merge`
> 
> Make sure to import this file instead of the default file in your Bootstrap imports

Reviewer: ```suggestion
Modify existing utilities with `map-get` and `map-merge`. Once again, be sure to import the utilities stylesheet before your custom utilities.
```

```scss

> @import "[PATH TO BOOTSTRAP]/scss/utilities";

Reviewer: ```suggestion
@import "bootstrap/scss/utilities";
```
